File: C++/ram.cpp
#include <string>

using namespace std;

class ram
{
    private:
        //atribut
        string kapasitas;
        int price;
    
    public:
        //konstruktor
        ram(){ }

        ram(string kapasitas, int price){
            this->kapasitas = kapasitas;
            this->price = price;
        }

        //prosedur setter data
        void setKapasitas(string kapasitas){
            this->kapasitas = kapasitas;
        }

        void setPrice(int price){
            this->price = price;
        }

        //prosedur getter data
        string getKapasitas(){
            return this->kapasitas;
        }

        int getPrice(){
            return this->price;
        }

        //destruktor
        ~ram(){ }
};

File: java/Ram.java
public class Ram {
    
    private String capacity;
    private int price;

    //konstruktor
    Ram(){ }

    Ram(String capacity, int price){
        this.capacity = capacity;
        this.price = price;
    }
    
    //prosedur setter data
    public void setCapacity(String capacity){
        this.capacity = capacity;
    }

    public void setPrice(int price){
        this.price = price;
    }

    //prosedut getter
    public String getCapacity(){
        return this.capacity;
    }

    public int getPrice(){
        return this.price;
    }

}
